### PR TITLE
Cleanup old endpoints that have been deprecated

### PR DESCRIPTION
**Pull request by rolfyone**
- v1/debug/states has been deprecated for a long time now, as has the v1/blocks endpoint.

 - v1/debug/heads seems to be a new deprecation so leaving that in the spec. I'm not sure why v2 exists, it looks like the execution_optimistic should have just been optional so that it was non breaking, but that horse hs bolted.

### CODE REVIEW

**Review by mpetrunic — APPROVED** [review]

File: beacon-node-oapi.yaml [hunk 1]
```diff
@@ -78,12 +78,8 @@ paths:
     $ref: "./apis/beacon/blocks/headers.yaml"
   /eth/v1/beacon/headers/{block_id}:
     $ref: "./apis/beacon/blocks/header.yaml"
-  /eth/v1/beacon/blocks:
-    $ref: "./apis/beacon/blocks/blocks.yaml"
```
Comment (michaelsproul) [inline beacon-node-oapi.yaml:82]: I don't think this should have been deleted. This API endpoint isn't deprecated and is the only way for validator clients to publish their non-blinded blocks to the beacon node

Comment (rolfyone) [reply beacon-node-oapi.yaml:82]: hrm i may have deleted the wrong thing? will check now...

Comment (rolfyone) [reply beacon-node-oapi.yaml:82]: you're correct, raising a pr to put back

Comment (michaelsproul) [reply beacon-node-oapi.yaml:82]: thanks!

**Pull request merged**